zanata-war/src/main/java/org/zanata/ZanataInit.java

```diff
@@ -112,8 +113,13 @@
     @In
     private EntityManagerFactory entityManagerFactory;
 
+    @In
+    private FacesTransactionEvents facesTransactionEvents;
+
     @Observer("org.jboss.seam.postInitialization")
     public void initZanata() throws Exception {
+        // disable the automatic JSF message when a transaction fails
+        facesTransactionEvents.setTransactionFailedMessageEnabled(false);
```

Comment:
Not sure whether this will work too (add this in component.xml)
```  <component class="org.jboss.seam.transaction.FacesTransactionEvents" installed="false" />```
Trying to reduce the seam stuff in our code base...